pxr/usdImaging/usdviewq/mainWindowUI.ui

```diff
@@ -2660,6 +2661,14 @@
     <string>Abstract Prims (Classes)</string>
    </property>
   </action>
+  <action name="actionShow_Prim_DisplayName">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>Use Display Name (Prims)</string>
+   </property>
+  </action>
```

Comment:
Can't immediately tell whether you did or not, but sometimes devs like to hand-edit the `.ui` files, which can cuase puzzling extraneous diffs for the next developer who actually uses `qdesigner`.  So we request that, if you **do** hand-edit a `.ui` file, you then open it in qdesigner and Save it back out so that Qt puts its own idea of ordering into the file.

🙏

Reply:
I probably did - I likely don't have the qdesigner installed but I'll take care of it.

Reply:
This one still needs to be addressed

Reply:
Saved from Qt Designer